frontend/src/components/utils/PathRoute.js

```diff
@@ -1,9 +1,9 @@
+import { useHistory } from "react-router-dom";
+
 const PathRoute = ({ path, children }) => {
-  var fullPath = window.location.href;
-  if (window.location.href.includes("?")) {
-    // Remove Query Params from the path
-    fullPath = window.location.href.split("?")[0];
-  }
+  const history = useHistory();
+  let fullPath = history.location.pathname;
```

Comment:
This willt work as expected. its because `history.location.pathnam`e does not include the hash (#) portion of the URL. In React Router, the pathname only contains the part of the URL before the #